cmd/juno-cli/cli/estimateFee.go

```diff
@@ -13,8 +13,25 @@ import (
 var estimateFeeCmd = &cobra.Command{
 	Use:   "estimate_fee CONTRACT_HASH FUNCTION_NAME INPUTS [SIGNATURE] [flags]",
 	Short: "Calculate transaction fee for calling a function.",
-	Long:  `See https://www.cairo-lang.org/docs/hello_starknet/cli.html#estimate-fee`,
-	Args:  cobra.MinimumNArgs(2),
+	Long: `See https://www.cairo-lang.org/docs/hello_starknet/cli.html#estimate-fee.
+
+	Please note that to estimate fee for functions with multiple inputs, you should use multiple -i flags with values, 
+	in the order of the necessary inputs. The network response will tell you that the number of inputs is wrong otherwise.
+	
+	Example with multiple inputs and expected response being returned:
+	
+	Input:
+	  juno-cli cx estimate_fee 0x0003a4d1be6ae6cccc7b6cc3cc16bbfd092c2a724ffe6be86c7b5b5fe6ec11d0 increase_decrease --network goerli -p -i 1 -i 2
```

Comment:
I think `cx` shouldn't be here.`juno-cli cx estimate_fee ...` doesn't work for me, `juno-cli estimate_fee ...` yes